res/values-pt-rBR/strings.xml

```diff
@@ -814,12 +814,12 @@
     <string name="Registrants">Registrados</string>
     <string name="Round_Of">Rodada De</string>
     <string name="Next_Tournament">Próximo Torneio</string>
-    <string name="Plasma_Won">Plasma Ganhado</string>
+    <string name="Plasma_Won">Plasma Ganho</string>
     <string name="Semi_Finals">Semi-Finais</string>
     <string name="Quarter_Finals">Quartas de Finais</string>
     <string name="Competing">Competindo</string>
     <string name="Finals">Finais</string>
-    <string name="Unregister">Cancelar Registro</string>
+    <string name="Unregister">Cancelar Inscrição</string>
```

Comment:
If you are trying to change for "Register" or "Unregister", this needs to be applied for every other line that contains the same text. Actually people who play tournaments are already used to "Registro", so please keep it as it was.